File: courses/codes/hw3-py-sol/hw3_q2.py
kg1 = int(input("Enter a first weight (kg): "))
g1 = int(input("Enter a first weight (g): "))
kg2 = int(input("Enter a second weight (kg): "))
g2 = int(input("Enter a second weight (g): "))

kg_total = kg1 + kg2
g_total = g1 + g2

if g_total >= 1000:
    kg_total = kg_total + g_total // 1000
    g_total = g_total % 1000

print("The sum is %dkg, %dg" % (kg_total, g_total))

File: courses/codes/java-class/hw3-java-sol/HW3Q2.java
import java.util.Scanner;

public class HW3Q2 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Enter first weight (in kg): ");
		int kg1 = sc.nextInt();
		System.out.print("Enter first weight (in g): ");
		int g1 = sc.nextInt();
		System.out.print("Enter second weight (in kg): ");
		int kg2 = sc.nextInt();
		System.out.print("Enter second weight (in g): ");
		int g2 = sc.nextInt();
		
		int kg = kg1 + kg2;
		int g = g1 + g2;
		
		if (g >= 1000) {
			kg = g / 1000;
			g = g % 1000;
		}
		
		System.out.println("The sum is " + kg + "kg, " + g + "g.");
	}

}
